src/firmware_new/boards/frankie_v1/frankie_v1.ioc

```diff
@@ -1,6 +1,8 @@
 #MicroXplorer Configuration settings - do not modify
-LWIP0.BSP.mode=
+<<<<<<< HEAD
+#LWIP0.BSP.mode=
 FREERTOS.Queues01=RobotLogProtoQ,16,TbotsProto_RobotLog,0,Dynamic,NULL,NULL
+>>>>>>> d3cb8535b42f4e950b7445c66243ea112370a42a
```

Comment:
this actually isn't an issue with your script, looks like cubemx doesn't give a crap and just runs even though the file is completely botched

try changing the numbers in the script